cmd/minikube/cmd/start_flags.go

```diff
@@ -580,6 +583,10 @@ func upgradeExistingConfig(cmd *cobra.Command, cc *config.ClusterConfig) {
 		cc.KubernetesConfig.NodePort = viper.GetInt(apiServerPort)
 	}
 
+	if cc.CertExpiration == 0 {
+		cc.CertExpiration = pkgutil.DefaultCertExpiration
```

Comment:
should this be in constant package instead of pkgutil ?

Reply:
good call, moved.